clap_mangen/tests/testsuite/roff.rs

```diff
@@ -105,3 +105,17 @@ fn value_name_without_arg() {
         cmd,
     );
 }
+
+#[test]
+fn flatten_help_false() {
```

Comment:
I added all relevant tests from `help.rs` and at least the recursive stuff isn't working right now. Is the recursive stuff strictly required for this PR to move forward?

Reply:
The recursive behavior is part of the definition of `flatten_help`.  We could possibly split it out but we'd need to be tracking it somehow.  Its likely best to go ahead and do that as it might affect the design in important ways